lib/qesapdeployment.pm

```diff
@@ -1277,6 +1277,7 @@ sub qesap_upload_crm_report {
     $log_filename =~ s/[\[\]"]//g;
 
     my $crm_log = "/var/log/$log_filename";
+    my $crm_log_postfix = is_sle('15+') ? 'tar.gz' : 'tar.bz2';
```

Comment:
Just an idea. Instead of having hard coded values ... I am thinking about doing something like:

```
my $upload_file = script_output("ls $crm_log*");
```
That should return full file path including correct extension.

Reply:
Dose here support `script_output` ?

Reply:
> Dose here support `script_output` ?

eventually yes `qesap_ansible_script_output`. Up to you if you like to use it or not.